src/core/usecases/catalogExplorer.ts

```diff
@@ -240,11 +239,11 @@ export const selectors = (() => {
         const catalog = state["~internal"].catalogs
             .filter(({ id }) => id === selectedCatalogId || state.search !== "")
             .map(catalog =>
-                catalog.catalog.packages.map(pack => ({
-                    "packageDescription": pack.description,
-                    "packageHomeUrl": pack.home,
+                catalog.catalog.entries.map(pack => ({
+                    "packageDescription": pack.latest().description,
+                    "packageHomeUrl": pack.latest().home,
```

Comment:
What you want here is a `catalogWithLatestCharts` selector that you reuse in `filteredSoftware`.  

Aslo, try co come up with consistent naming for things.  `pack` isn't ideal `entries.map(entry=>`

Reply:
Now, the data is sorted. So we can access the latest version by getting the first element